### ISSUE TITLE
Timestamp in video is not internationalised

### ISSUE THREAD

**Issue by bigjools** [ehendrix23/tesla_dashcam#109]
The timestamp appears to be fixed to MM/DD/YY, which is very confusing if you're not in the USA. Can you either change it to YYY/MM/DD, use the current locale, or provide an option.

**Comment by ehendrix23** [comment]
Do you mean the timestamp displayed within the video? It is defined based on the current locale.

Following is the entry that sets the text and how to display it (epoch_time is replaced then with actual timestamp for video):
   '%{pts\:localtime\:{epoch_time}\:%x %X}

The last argument defines how the timestamp should be represented, it is based on strftime:
       %x     The preferred date representation for the current locale
                 without the time.
       %X     The preferred time representation for the current locale
                  without the date.

**Comment by bigjools** [comment]
My locale was set to my Australian locale at the time, which is definitely not how it came out in the final video. Is there any other debugging I can look into? I'm not familiar with the code yet but I can dive in later.

**Comment by ehendrix23** [comment]
Using --loglevel DEBUG will provide more information on things going on, including the FFmpeg command that is executed. The timestamp is put in there with the drawtext parameter.

**Comment by bigjools** [comment]
Hi - sorry for the delay in replying. I have been playing around with this. The debug log confirms that the timestamp is as you say and I can see the line in the code that sets it. I've tried to set the TZ and LC_TIME environment variables and it still does the date in US format.

If I remove the `%x %X` then I get the YYYY-MM-DD format that I want. The documentation seems to suggest that just using localtime is enough to do the date in the local format. Can you confirm for your locale? If so, we can just remove the formatting directives.

**Comment by neg2led** [comment]
I'm seeing this issue as well, on a Windows Server 2016 box. Global culture (base Windows version) is set to en-GB, local user/display culture is en-AU, both of which should present a DD/MM/YYYY - but I'm getting MM/DD/YYYY stamped into the videos.

I suspect this is a side-effect of using the prepackaged Windows binary rather than running the Python script directly... How is this packaged together? Maybe TZ/LC_TIME environment variables end up baked into the package? Can I override these at the commandline?

Here's a (poorly-)formatted example of the debug output for the ffmpeg command:

```
2020-08-08 20:03:38,122:DEBUG:  __main__        FFMPEG Command: 
['C:\\Users\\aholmes\\AppData\\Local\\Temp\\2\\_MEI136642\\ffmpeg.exe
-loglevel error
-i Q:\\temp\\TeslaCam\\SavedClips\\2020-03-06_11-46-57\\2020-03-06_11-37-43-left_repeater.mp4
-i Q:\\temp\\TeslaCam\\SavedClips\\2020-03-06_11-46-57\\2020-03-06_11-37-43-front.mp4 
-i Q:\\temp\\TeslaCam\\SavedClips\\2020-03-06_11-46-57\\2020-03-06_11-37-43-right_repeater.mp4 
-i Q:\\temp\\TeslaCam\\SavedClips\\2020-03-06_11-46-57\\2020-03-06_11-37-43-back.mp4
-filter_complex "
color=duration=59.83:s=3840x2016:c=black, fps=30 
[base];
[0:v] setpts=PTS-STARTPTS, scale=1280x960 , hflip [left];
[1:v] setpts=PTS-STARTPTS, scale=1280x960  [front];
[2:v] setpts=PTS-STARTPTS, scale=1280x960 , hflip [right];
[3:v] setpts=PTS-STARTPTS, scale=1280x960 , hflip [rear];
[base][left] overlay=eof_action=pass:repeatlast=0:x=0:y=528 [left1];
[left1][front] overlay=eof_action=pass:repeatlast=0:x=1280:y=0 [front1];
[front1][right] overlay=eof_action=pass:repeatlast=0:x=2560:y=528 [right1];
[right1][rear] overlay=eof_action=pass:repeatlast=0:x=1280:y=1056 [rear1];
[rear1] drawtext=fontfile=/Windows/Fonts/arial.ttf:fontcolor=white:fontsize=96:borderw=2:bordercolor=black@1.0:x=(w/2-text_w/2):y=(960 + 5):text='%{pts\\:localtime\\:1583455063\\:%x %X}' [tmp0]"
-map [tmp0]
-preset medium
-crf 28
-c:v libx265
-metadata description=Created by tesla_dashcam v0.1.16
-y D:\\temp\\TeslaCam\\2020-03-06_11-37-43.mp4
```

**Comment by bigjools** [comment]
@neg2led If you try my suggestion on https://github.com/ehendrix23/tesla_dashcam/issues/109#issuecomment-581801092 to remove some of the formatting options, does it work for you? FWIW I am using this on Linux so I doubt it's a Windows problem, probably an ffmpeg bug or documentation problem.

**Comment by neg2led** [comment]
I can't particularly easily check, since I'm not sure how the Windows binary is packaged together - I've no doubt it would work if I was running the python script directly, but i'm trying to avoid installing Python on the system in question if I can!

**Comment by ehendrix23** [comment]
Not sure why not localized, but in v0.1.17 you can now provide your own timestamp format hence it should fix this issue.
Beta just been released: https://github.com/ehendrix23/tesla_dashcam/releases/tag/v0.1.17b1

**Comment by ehendrix23** [comment]
Closing as this issue has been fixed in v0.0.17 beta release, hopefully soon to be released as final
